### ISSUE TITLE
Add the 'NVDA Sounds Volume' slider to Synth Settings Ring, displays only when 'follow speech volume' is disabled

### ISSUE THREAD

**Issue by valiant8086** [nvaccess/nvda#15063]

**Comment by valiant8086** [comment]
Sorry I hit enter when I didn't mean to and broke the template. I also thought there was a separate section for feature requests but this looks for all the world like a bug report.

I'm thinking it might be a good idea to add the brand new NVDA sounds volume slider to synth settings ring. If so, it would only be there when the check box to follow speech volume that is currently in advanced settings was unchecked.

People may be against this though. I think I'd never actually see it because I just want it to follow my speech volume anyway.

**Comment by CyrilleB79** [comment]
Hi @valiant8086 

There is actually a specific template for new feature requests. Go to the [new issue](https://github.com/nvaccess/nvda/issues/new/choose) page, and in the section "Feature request", click the link "Get started".

You may copy the content of the template in the initial description of this issue. For this, in the current issue:
* go to the first level 3 heading, which correspond to your first comment
* press upArrow to land on the "Show options" button
* press this button and choose "Edit comment"
* you can then edit your initial comment and validate.

**Comment by leonardder** [comment]
While I really like this suggestion and also thought about this myself, implementing this is a whole lot harder than you might expect. The synth settings ring contains a lot of magic to operate settings from the synthesizer instance. These settings are in a different area than the sounds volume slider setting

**Comment by CyrilleB79** [comment]
If this request is accepted, the decision should be made/accepted by NVAccess to convert the "synth settings ring" into a "settings ring". This is not just a change of name but a whole redesign of the code as explained by @leonardder.

In this case the question becomes much broader: which setting can/should be in the ring? Can the list of settings be customized? May we have various settings rings for various sets of settings?

**Comment by CyrilleB79** [comment]
Also, I am not in favor of including such an easy access to NVDA sound while this parameter is in Advanced settings.

Since modifying a value in Advanced settings requires to check the checkbox "I understand", this permission should not be bypassed.

**Comment by valiant8086** [comment]
@CyrilleB79 can't argue with your thoughts about not doing this while that setting is under Advanced. Think I read in the main discussion about the wasapi changes that are in advanced would be moved out of advanced, although I can't remember if there was a planned timing for that. Might be waiting on fixes to being unable to go to sleep and such.

Did anyone already create any issue about rewriting the backend that the synth settings ring uses then? Perhaps, since it's working as is, it's going to be a hard sell to get someone on board to do the work on that. However, if it happened, I could picture a settings page with check boxes to enable or disable the different things that can be adjusted, so that users can take out things they didn't want and add seemingly obscure ones that most others wouldn't.

**Comment by XLTechie** [comment]
I agree with @valiant8086. It would make it more like the iOS rotor, which is 
quite useful and configurable.

That said, you won't catch me trying to implement that any time soon. :)

**Comment by michaelDCurran** [comment]
Based on the above comments such as the fact that the setting is in advanced settings, it is techhnically not quite a synth parameter, and that the reporter seems to have said that'd probably never see it anyway, and that NV Access is not relaly motivated right now to convert Synth settings ring into a more generalized feature, I'm going to close this.
However, I would encourage someone to file an issue on converting synth settings ring to a more generalized feature.
